src/pkgcore/ebuild/domain.py

```diff
@@ -44,6 +44,7 @@
 from . import const
```

Comment:
Commit message has a typo (extra 'package/')

Reply:
fixing shortly, thanks for the spot.